### ISSUE TITLE
Add page UI options shouldn't be set back , padding missing on smaller viewports 

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-cms#2372]
The page types within the add page UI shouldn't have a negative margin on the left, on smaller viewports these page options touch the CMS menu.
<img width="439" alt="pasted_image_28_01_19__12_59_pm" src="https://user-images.githubusercontent.com/555033/51808861-c396fa80-22fe-11e9-9a70-83234dee1433.png">

CSS to remove:
![image](https://user-images.githubusercontent.com/555033/51808870-e1fcf600-22fe-11e9-940c-6df2a00c9aff.png)

**Comment by maxime-rainville** [comment]
Just had a quick look. If the left/right margin can remove, the step 2 option get misaligned relative to the step 1.

![image](https://user-images.githubusercontent.com/1168676/52038606-a03aac80-2597-11e9-8e9b-4f0dd7f3e3e0.png)

**Comment by clarkepaul** [comment]
Yeah I saw that, I thought it was the leaser of two evils. Ideally we would remove the padding around page selection of pages and revert back to the default checkbox style.

A side note is that we would eventually redesign this experience to make adding pages possible directly from the site tree rather than having to navigate away (similar to the add block functionality of Elemental), although that will be a way off development wise.

**Comment by ScopeyNZ** [comment]
Would that use the same design pattern as elemental? Using [`PopoverOptionSet`](https://silverstripe.github.io/silverstripe-admin/?knob-status=default&knob-maxSize=200&knob-tags=%5B%7B%22key%22%3A%22justKey%22%7D%2C%7B%22key%22%3A%22KeyPair%22%2C%22value%22%3A%22123%22%7D%2C%7B%22key%22%3A%22KeyLabel%22%2C%22label%22%3A%22Some%20label%22%7D%2C%7B%22key%22%3A%22KeyPairLabel%22%2C%22label%22%3A%22Some%20label%22%2C%22value%22%3A%22123%22%7D%5D&knob-pill=true&knob-message=Hello%20World&knob-value=&knob-focusable=true&knob-label=&knob-dataKey=HelloWorld&knob-Search=true&selectedKind=Admin%2FPopoverOptionSet&selectedStory=Simple%20Example&full=0&addons=1&stories=1&panelRight=0&addonPanel=kadira%2Fjsx%2Fpanel)?

**Comment by clarkepaul** [comment]
Probably @ScopeyNZ , your favourite component :)
